3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -276,6 +276,33 @@ I parametri modificabili sono gli stessi che vengono richiesti durante la creazi
 
 Successivamente all'inserimento dei dati rappresentativi della zona modificata, è possibile premere sul pulsante "Salva le modifiche alla Zona" per aggiornare, come richiesto, l'elemento 3D corrispondente nell'ambiente di lavoro.
 
+== Richiesta spostamento prodotto
+
+#figure(
+    grid(
+        columns: 2,
+        rows:    (auto, auto),
+        [ #image("./imgs/spostamento_drag_and_drop.png", width: 80%)],
+        [ #image("./imgs/spostamento_non_collocato.png", width: 80%)],
+    ),caption: [Spostamento prodotti con drag and drop (sinistra), menù spostamento prodotti non collocati (destra)],
+) <spostamento>
+
+=== Spostamento tramite drag and drop
+
+Lo spostamento tra due posizioni di un prodotto avviene tramite drag and drop.
```

Comment:
Cambierei in: "Lo spostamento di un prodotto da un bin a un altro avviene tramite _drag and drop_."